workspaces/ui-v2/src/components/ShapeRenderer/ShapePrimitive.tsx

```diff
@@ -1,34 +1,27 @@
 import * as React from 'react';
 import makeStyles from '@material-ui/styles/makeStyles';
 import { useSharedStyles } from './SharedStyles';
-import { IShapeRenderer, JsonLike } from './ShapeRenderInterfaces';
-import { useShapeRenderContext } from './ShapeRenderContext';
+import { IShapeRenderer, JsonLike } from '<src>/types';
 import classNames from 'classnames';
 
-export const ShapePrimitiveRender = ({ jsonType, value }: IShapeRenderer) => {
+export const ShapePrimitiveRender = ({ jsonType }: IShapeRenderer) => {
```

Comment:
Value was always undefined - this is something we'll need to add back for show examples, but given that value wasn't typed consistently - it'll be easier to reimagine + update typing where appropriate when we come to implement that feature